homeassistant/components/tellduslive.py

```diff
@@ -56,7 +57,8 @@ def setup(hass, config):
         return False
 
     hass.data[DOMAIN] = client
-    client.update(utcnow())
+
+    hass.bus.listen(EVENT_HOMEASSISTANT_START, client.update)
```

Comment:
Good point. Keep the start listener and start polling on START event. Append in added_to_hass hook